src/nav/dashboards.yml

```diff
@@ -77,6 +77,8 @@ pages:
             path: /docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/nested-aggregation-make-ordered-computations-single-query
           - title: How funnels work
             path: /docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/funnels-evaluate-data-series-related-events
+          - title: Query data from lookup tables
```

Comment:
Should the title here and the title shown on the page be the same ("Use lookup table data in queries")?  It seems like it would help with navigation.

I'd also vote for placing the lookup tutorial right after the "NRQL Subquery joins" tutorial in the nav, since they're closely-linked features.